adam-cli/src/test/scala/org/bdgenomics/adam/cli/ADAM2FastaSuite.scala

```diff
@@ -17,18 +17,16 @@
  */
 package org.bdgenomics.adam.cli
 
-import com.google.common.io.Files
 import java.io.File
-import org.apache.spark.rdd.RDD
-import org.bdgenomics.adam.rdd.ADAMContext._
+
```

Comment:
> Also I'm not sure if I can change that IntelliJ setting per-project :( but lmk if given all this you still want to just have "one import blob" be the rule.

That would be my preference for consistency with the remainder of the codebase. If it's a big hassle from your side, I can fix on merge.